release-notes.md

```diff
@@ -50,14 +20,65 @@ To test out the beta version for yourself,
 follow these [instructions](https://docs.timescale.com/clustering/getting-started/scaling-out).
```

Comment:
What do you think about pointing to blog instead?  The connected docs don't really give a picture of the architecture & why you'd use -- a bit low level?

Reply:
Namely, both of the articles, maybe as a bulleted list or something?

- https://blog.timescale.com/blog/building-a-distributed-time-series-database-on-postgresql/
- https://blog.timescale.com/blog/achieving-optimal-query-performance-with-a-distributed-time-series-database-on-postgresql/

Reply:
@mfreed are you ok publishing for now as is and let me update this next week (I agree with the suggestion I am just a little short on time but will do it over the weekend) but was hoping to push docs today just so they are up there and match the fact that the release is out  OK by you?